src/cljs/main/broadfcui/components/workspace_selector.cljs

```diff
@@ -17,15 +16,13 @@
      (let [{:keys [selected-index]} @state]
        (case selected-index
          0 ["No workspace selected"]
-         1 ((:new-workspace-form @locals) :validate)
          nil)))
    :get-selected-workspace
    (fn [{:keys [state locals]}]
      (let [{:keys [workspaces selected-index]} @state]
        (case selected-index
          0 {:error "No workspace selected"}
-         1 {:new-workspace ((:new-workspace-form @locals) :get-field-values)}
-         {:existing-workspace (nth workspaces (- selected-index 2))})))
+         {:existing-workspace (nth workspaces (- selected-index 1))})))
```

Comment:
Note: if users wish to export a method to a new workspace, they will have to first create it in Terra UI. I have tested out the workflow on this branch, and it does succeed (populating a workspace that I created in Terra UI Dev).